packages/opds-web-client/src/components/Search.tsx

```diff
@@ -29,6 +29,7 @@ export default class Search extends React.Component<SearchProps, {}> {
             <input
               className="form-control"
               ref="input"
+              aria-label="Enter search keyword(s)"
               type="text"
```

Comment:
The `(s)` seems annoying if you're using a screen reader.  "Enter search keyword or keywords"?